src/protocolsupport/protocol/packet/middle/serverbound/play/MiddleStopSound.java

```diff
@@ -13,6 +13,8 @@ public MiddleStopSound(ConnectionImpl connection) {
 
 	//TODO: structure
 	protected ByteBuf data;
+	protected String name = "";
```

Comment:
same here, but this class actually doesnt have a PE subclass yet- so you'll need to add it (if PE has an equivalent)